apps/opik-frontend/src/lib/sorting.ts

```diff
@@ -38,3 +38,13 @@ export const mapFeedbackScoresColumn = (column: ColumnSort): ColumnSort => {
   }
   return column;
 };
+
+export const mapUsageColumn = (column: ColumnSort): ColumnSort => {
+  if (column.id.startsWith("usage_")) {
+    return {
+      ...column,
+      id: column.id.replace("usage_", "usage."),
+    };
+  }
+  return column;
+};
```

Comment:
It's best to place this function above processSorting, as using variables or functions before they are defined can sometimes cause build issues. The rest seems fine